src/components/atoms/ToggleRP/ToggleRP.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+
+import * as classNames from 'classnames'
+import { ClassNameProps } from 'interfaces/Props'
+import './ToggleRP.scss'
+
+interface ToggleProps extends ClassNameProps {
+  readonly children?: any
+  readonly disabled?: boolean
+  readonly on?: boolean
+  readonly onClick?: () => void
+}
+
+interface ToggleState {
+  readonly on?: boolean
+}
+
+const callAll = (...fns: ReadonlyArray<(...args: ReadonlyArray<any>) => void>) => (...args: ReadonlyArray<any>) =>
```

Comment:
This is what is breaking the Netlify build. How can we correctly do the Typescript typing for this function? Array is converted to ReadonlyArray to pass the lint rules, but doesn't work with rest parameters.